packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -46,10 +58,7 @@ export default connectSuggestedActions(
         ) =>
           <SuggestedAction
             buttonText={
-              type === 'messageBack' ?
-                title || displayText
-              :
-                title || (typeof value !== 'string') ? JSON.stringify(value) : value
+             renderSuggestedAction({title, type, value})
```

Comment:
A comment about the naming of this function: when I see `renderBlah` I expect that function to return some jsx. In this case it just returns a string, so I would suggest renaming to something like `suggestedActionText`.